### ISSUE TITLE
error while splitting into chunks

### ISSUE THREAD

**Issue by dnsorozco7** [TomFrankly/pipedream-notion-voice-notes#39]
Error — An error occured while attempting to split the file into chunks, or while sending the chunks to OpenAI.

**Comment by VirtualKenji** [comment]
Same.

I tried:
- using google drive
- using dropbox
- using original test audio file
- using a new .wav file
- using new Open AI API's
- Re-uploading / re-downloading audio files

**Comment by TomFrankly** [comment]
@dnsorozco7 @VirtualKenji can you both try this fix? https://thomasjfrank.com/how-to-transcribe-audio-to-text-with-chatgpt-and-notion/#openai-error

If that doesn't work, please post the contents of your Logs tab on the failed workflow so we can use that info to debug.

**Comment by weseeclearly** [comment]
Heya, I'm also having the same error. It worked for me already twice, but this afternoon when I tried again I got the title error. I just tried generating another API key (been a paying user of openai for 4 months now, so didn't switch account types), and that didn't fix it.

Here is my log:

![image](https://github.com/TomFrankly/pipedream-notion-voice-notes/assets/78986062/37e4613c-a77d-47df-9ba5-6c2fa4e0eb4e)
